src/stories/Library/autosuggest-material/autosuggest-material.scss

```diff
@@ -37,23 +38,14 @@
       background-color: $c-global-secondary;
     }
 
-    @include breakpoint-m {
-      width: 33.3%;
-    }
-
     &__content {
-      flex-flow: nowrap;
-      display: flex;
-      flex-direction: row;
+      display: grid;
+      gap: 15px;
+      grid-template-columns: 71px 1fr;
       width: 95%;
```

Comment:
Consider if this change is needed.

The original approach seems more flexible. There is no reference to specific value such as 71px.

Reply:
This is needed because we now have different cover aspects. But I have changed the gird size for the cover to use the nearest global variable

Reply:
This is needed because we now have different cover aspects. But I have changed the gird size for the cover to use the nearest global variable